### ISSUE TITLE
Ensure that text and background colors have enough contrast on Show Pages

### ISSUE THREAD

**Issue by christhepianist** [samvera/hyku#3054]
vpat.hykudemo.org
Version 7.1

### Descriptive summary, including screenshots, if any
Color contrast on this type of page does not meet the minimum accessibility standard.
Increase the color contrast to 4.5:1 or higher.
Currently, the contrast is 4.16 : 1
Color details:
- Text color: #2e74b2 / RGB (46, 116, 178)
- Background color: #e9ecef / RGB (233, 236, 239)
Specific areas to be fixed:
- Breadcrumbs: Home, Dashboard, Works, <title of the work>
- Thumbnail title 

### Rationale
Text and background colors that conflict or have too little contrast between each other can cause readability issues for users who are color blind or have low vision (vision loss equivalent to approx. 20/40 vision).

### Expected behavior
Color contrast is at least 4.5:1

### Actual behavior
Color contrast is 4.16:1.

### Steps to reproduce the behavior
Use SiteImprove on this type of page to see the error.

### Related work (other tickets, possibly the same issue in Hyrax)

<img width="1897" height="1050" alt="Image" src="https://github.com/user-attachments/assets/de83af59-5552-4980-aa94-f7d2c8b90202" />

<img width="1520" height="472" alt="Image" src="https://github.com/user-attachments/assets/83677528-37dd-41ca-9187-cb22aa282418" />
